--- epm-3.7/rpm.c~	2003-10-01 19:27:15.000000000 +0000
+++ epm-3.7/rpm.c	2018-08-23 14:26:52.168263293 +0000
@@ -106,7 +106,6 @@
   fprintf(fp, "Name: %s\n", prodname);
   fprintf(fp, "Version: %s\n", dist->version);
   fprintf(fp, "Release: %d\n", dist->relnumber);
-  fprintf(fp, "Copyright: %s\n", dist->copyright);
   fprintf(fp, "Packager: %s\n", dist->packager);
   fprintf(fp, "Vendor: %s\n", dist->vendor);
   fprintf(fp, "BuildRoot: %s/buildroot\n", absdir);
